RELEASE_NOTES.rst

```diff
@@ -1,3 +1,52 @@
+coala 0.9.0
+===========
+
+Command Line Interface Changes:
+
+- Users can press enter to dismiss a result by default.
+- A lot more strings, like ``roger`` or ``no way`` are allowed for boolean
+  values. (https://github.com/coala/coala/commit/728b7b02da8ca8f91b91c67784872244c0820a77)
+- The section name is now displayed when asking the user for missing settings.
+- Minor performance improvements could be implemented.
```

Comment:
sounds unimportant. Maybe "~~Minor~~ Some performance improvements could be ~~implemented~~ achieved"